### PR TITLE
feat: create image docker for backend

### PR DESCRIPTION
**Pull request by kellyCSalazar**
## Describe your changes
Create image docker for backend

## Issue ticket number and link
https://trello.com/c/nJvEpg4O

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Review by SilviaTrujillano23 — APPROVED** [review]

**Review by anversa-pro — APPROVED** [review]
Yay! habemus docker!!

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
@kellyCSalazar please could you describe why yarn.lock was removed?
on the other hand, why a library or a helper is part of this PR?

File: helpers/utils.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const isObjectEmpty = data => (Object.entries(data).length === 0 ? true : false)
```
Comment (FelipeLosada1996) [inline helpers/utils.js:1]: This file isn't corresponding to this MR, and was already added

**Comment by kellyCSalazar** [comment]
Those changes were from the previous pr, both the yarn and the helpers folder.
A rebase was made to fix it

File: helpers/utils.js (hunk 1 above)
Comment (kellyCSalazar) [reply helpers/utils.js]: Those changes were from the previous pr, both the yarn and the helpers folder.
A rebase was made to fix it

**Review by jlora-mck — APPROVED** [review]

**Review by FelipeLosada1996 — APPROVED** [review]

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**